arcgis-ios-sdk-samples/Layers/OpenStreetMap layer/README.md

```diff
@@ -10,12 +10,12 @@ You may want to create a map with an [OpenStreetMap](https://www.openstreetmap.o
 
 ## How to use the sample
 
-When the sample opens, it will automatically display the map with the OpenStreetMap basemap. Pan and zoom to observe the basemap.
+The OpenStreetMap basemap will display upon launch. Pan and zoom to explore the basemap.
 
 ## How it works
 
-1. Create an `AGSMap` with `osmStandard` as the `basemapStyle` and specify the other properties.
-2. Apply the `AGSMap` to the `AGSMapView`.
+1. Create an `AGSOpenStreetMapLayer`, and set it as the `AGSMap`'s basemap.
+2. Set the `AGSMap` to the `AGSMapView`.
```

Comment:
The layer is used as the base layer, not the base map. Also, it isn't set on the map; it is provided as initialization (creation) time.

```suggestion
1. Create an instance of `AGSOpenStreetMapLayer`.
2. Create an instance of `AGSMap` using the OpenStreetMap layer as the base layer.
2. Set the map to the map view's `map` property.
```